### DISCUSSION TITLE
[Esame 16 Giugno 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#359] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8e675069-480a-4a43-9d5c-6f11bc13fe0e)

**Comment by Traster-uni** [comment]
![soluz_es1_esameGiugno_ProfBandiera](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84441861/9eadce1e-4e0b-4b57-9428-0ef2f80cdb2d)

**Comment by Frisbon** [comment]


**Reply by Traster-uni** [reply]
Alla prima congruenza del sistema 10x = 6 (mod 16) dividendo tutto per 2 verrebbe 5x = 3 (mod 8) non 5x = 6 (mod 8)

**Reply by Frisbon** [reply]
Grazie mille! Ora ri-sistemo
